docs/ref/models/fields.txt

```diff
@@ -1116,7 +1116,7 @@ Django also defines a set of fields that represent relations.
 ``ForeignKey``
 --------------
 
-.. class:: ForeignKey(othermodel, [**options])
+.. class:: ForeignKey(othermodel, on_delete, [**options])
```

Comment:
@carljm is https://github.com/fcurella/django/commit/e6d393b962b52579cfd23d5a42ce47854f252544 close to what you had in mind?